.github/workflows/update_pardoks.yml

```diff
@@ -0,0 +1,18 @@
+name: Apply PARDOK XML Diff to DB
+
+on:
+  workflow_dispatch: {}
+  repository_dispatch:
+    types: [pardok_diff_apply]
+
+jobs:
+  pardok-xml-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Apply PARDOK XML Diff
+        run: |
+          npm ci
+          curl https://www.parlament-berlin.de/opendata/pardok-wp19.xml > pardok-wp19.xml
```

Comment:
What if the curl command fails? Shouldn't we do a curl "https://..." && tsx ...

Reply:
changed